### PR TITLE
{tools,vis}[GCCcore/13.2.0] Ghostscript v10.02.1, GTK3 v3.24.39

### PR DESCRIPTION
**Pull request by SimonPinches**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/g/Ghostscript/Ghostscript-10.02.1-GCCcore-13.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+version = '10.02.1'
+
+homepage = 'https://ghostscript.com'
+description = """Ghostscript is a versatile processor for PostScript data with the ability to render PostScript to
+ different targets. It used to be part of the cups printing stack, but is no longer used for that."""
+
+toolchain = {'name': 'GCCcore', 'version': '13.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs10021']
```
Comment (akesandgren) [inline easybuild/easyconfigs/g/Ghostscript/Ghostscript-10.02.1-GCCcore-13.2.0.eb:13]: Could still have used the style that 10.0.0 easyconfig uses,
```
source_urls = [
    'https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs%s/' % version.replace('.', ''),
]
```

**Comment by akesandgren** [comment]
@boegelbot Please test @ generoso

File: easybuild/easyconfigs/g/Ghostscript/Ghostscript-10.02.1-GCCcore-13.2.0.eb (hunk 1 above)
Comment (SimonPinches) [reply easybuild/easyconfigs/g/Ghostscript/Ghostscript-10.02.1-GCCcore-13.2.0.eb]: Thanks @akesandgren, I've now included this change.

**Comment by akesandgren** [comment]
@boegelbot Please test @ generoso

**Review by akesandgren — APPROVED** [review]
LGTM

**Comment by akesandgren** [comment]
Going in, thanks @SimonPinches!

**Pull request merged**